Remotion/Web/Core/UI/Controls/WebTreeNodeCollection.cs

```diff
@@ -23,6 +25,24 @@ namespace Remotion.Web.UI.Controls
   /// <summary> A collection of <see cref="WebTreeNode"/> objects. </summary>
   public class WebTreeNodeCollection : ControlItemCollection
   {
+    /// <summary>
+    /// Groups the top level <see cref="WebTreeNode"/>s by their <see cref="WebTreeNode.Category"/>
+    /// </summary>
+    /// <param name="nodes">The nodes of the <see cref="WebTreeView"/></param>
+    /// <returns></returns>
+    public static IReadOnlyList<WebTreeNode> GroupByCategory (WebTreeNode[] nodes)
```

Comment:
we don't need to make the parameter into an array, we can stick with WebTreeNodeCollection here. simplifies things at the call site.